README.md

```diff
@@ -50,3 +50,21 @@ Infrastructure and code are deployed to AWS using Serverless.
 ### Automated deployments
 Merging into `master` triggers an automated deployment into the staging environment.
 To promote this to production you will need to manually approve the deployment in CircleCI.
+
+## Pages and API endpoints
+Login is handled by Single View, so long as the Hackney token cookie is set users will be signed in here too.
+
+### GET /api/snapshots/{id}
+Retrieves a vulnerabilities snapshot, given a snapshot id.
+
+### POST /api/snapshots/find
+Used by Single View to find snapshots related to a particular person given their name, an optionally an array of identifiers.
```

Comment:
Is it okay to bind it to SV? What if it's reused by something else in the future? 🤔

Reply:
Fair point, I think it's worth adding in that it is used by Single View at the moment - since it's not used by anything else in this project. We can always update it if other things start using that endpoint. Maybe we could reword it though?

```suggestion
Used by other applications (such as Single View) to find snapshots related to a particular person given their name, an optionally an array of identifiers.
```

Do you think that works better?

Reply:
yeah, looks good!